### DISCUSSION TITLE
Can I tell if user is on a (video) call?

### DISCUSSION THREAD

**Discussion by mdrichardson** [timmo001/system-bridge#405] [Ideas] [answered]
Is it possible to tell if the system's current user is in a video call? I couldn't see anything in the API that seems to get information related to this, so this may be more of a feature request. Ideally, I could get information like:

1. System notification are blocked or on DnD
2. WebCam is active
3. Microphone is active
4. App in focus (Teams, Slack, Zoom, etc)

[Hass Workstation Service](https://github.com/sleevezipper/hass-workstation-service) does something like this, but I'd love to just use a single app.

**Comment by timmo001** [comment, marked as answer]
See #1221
